docs/WithAwsPlugin.md

```diff
@@ -60,3 +60,15 @@ validate.then(deployQa)
         .then(deployProd)
         .build()
 ```
+
+If you want to specify a role session duration other than the default of 1 hour (3600 seconds), you can do so by providing an integer `duration` parameter to the `withRole()` call. You _must_ specify the Role ARN in this case, even if `null`. i.e.:
+
+```
+WithAwsPlugin.withRole('MY_ROLE_ARN', 43200).init()
+```
+
+or, to use the default environment variables for the role ARN:
+
+```
+WithAwsPlugin.withRole(null, 43200).init()
+```
```

Comment:
This fix looks good overall, but one minor tweak.  

This feature is an example of trying to configure a plugin that has multiple different parameters, which can be set independently of one another. 

Rather than trying to overload the `.withRole()` method to optionally accept a role/role+duration/duration, can we simply add a `.withDuration()` method?  That way, if you want to set duration, you can do so explicitly.  If you want to set the role, you can do so explicitly.  If you want to set them both, then you chain and call them both explicitly.

We can avoid the awkardness and lack-of-context in your example above: `WithAwsPlugin.withRole(null, 43200)`.  "What is 43200?  What does that have to do with a null role?"  vs `WithAwsPlugin.withDuration(43200)` which gives context to 43200.  "43200 is probably a duration".

This is a convention that's used heavily across all the plugins, so I'd like to keep that consistent.  https://en.wikipedia.org/wiki/Fluent_interface